krod/interfaces/cli.py

```
@@ -63,11 +63,15 @@ def process_command(self, command: str) -> bool:
+                # args = command.split()[1:]
+                import shlex
+                args = shlex.split(command)[1:]
                 # Create a standalone context and invoke the command
                 ctx = vector_store_commands.make_context('vector_store', args)
-                vector_store_commands.invoke(ctx)
-                return True
+                # vector_store_commands.invoke(ctx)
+                # return True
+                exit_code = vector_store_commands.invoke(ctx)
+                return exit_code == 0
             except click.ClickException as e:
```

Ruff (0.11.9)

Undefined name `click`

(F821)